<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chatroom</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="list_users">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>160</width>
      <height>571</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_msg">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>450</y>
      <width>570</width>
      <height>90</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_send">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>540</y>
      <width>91</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_history">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>570</width>
      <height>451</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lab_user">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>550</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_username">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>550</y>
      <width>260</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_look_key">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>540</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>查看共有秘钥</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
